abb_bringup/launch/abb_moveit.launch.py

```diff
@@ -32,94 +30,47 @@ def launch_setup(context, *args, **kwargs):
     moveit_config_package = LaunchConfiguration("moveit_config_package")
     moveit_config_file = LaunchConfiguration("moveit_config_file")
 
-    # Planning context
-    robot_description_content = Command(
-        [
-            PathJoinSubstitution([FindExecutable(name="xacro")]),
-            " ",
-            PathJoinSubstitution(
-                [FindPackageShare(support_package), "urdf", robot_xacro_file]
-            ),
-        ]
-    )
-    robot_description = {"robot_description": robot_description_content}
-
-    robot_description_semantic_content = Command(
-        [
-            PathJoinSubstitution([FindExecutable(name="xacro")]),
-            " ",
-            PathJoinSubstitution(
-                [FindPackageShare(moveit_config_package), "config", moveit_config_file]
-            ),
-        ]
+    # MoveIt configuration
+    moveit_config = (
+        MoveItConfigsBuilder("abb_irb1200_5_90")
+        .robot_description(file_path=os.path.join(
+            get_package_share_directory(f'{support_package.perform(context)}'),
+            'urdf', f'{robot_xacro_file.perform(context)}'))
+        .robot_description_semantic(file_path=os.path.join(
+            get_package_share_directory(f'{moveit_config_package.perform(context)}'),
+            'config', f'{moveit_config_file.perform(context)}'))
+        .planning_pipelines(
+            pipelines=["ompl"],
+            default_planning_pipeline="ompl",)
+        .robot_description_kinematics(file_path=os.path.join(
+            get_package_share_directory(f'{moveit_config_package.perform(context)}'),
+            'config', 'kinematics.yaml'))
+        .trajectory_execution(file_path=os.path.join(
+            get_package_share_directory(f'{moveit_config_package.perform(context)}'),
+            'config', 'moveit_controllers.yaml'))
+        .planning_scene_monitor(True, True, True, True, True, True)
```

Comment:
If you prefer leaving this in, suggest to pass the argument names for clarify
 ```suggestion
        .planning_scene_monitor(
            publish_planning_scene=True,
            publish_geometry_updates=True,
            publish_state_updates=True,
            publish_transforms_updates=True)
```